packages/api/app/distribution.js

```diff
@@ -62,7 +63,14 @@ distributionApp.use((err, req, res, _next) => {
 
 const server = awsServerlessExpress.createServer(distributionApp);
 
+const handler = (event, context) => {
+  if (event.eventType === 'createBucketMapCache') {
```

Comment:
The approach here is to use the handler instead of posting to an express endpoint, which seems fine given the nature of the distribution app..... 👍🏻